src/proxy-ui-api/frontend/src/views/InitialConfiguration/InitialConfiguration.vue

```diff
@@ -0,0 +1,231 @@
+        }}</v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step :complete="currentStep > 3" step="3">{{
+          $t('initialConfiguration.pin.title')
+        }}</v-stepper-step>
+      </v-stepper-header>
+
+      <v-stepper-items v-if="isAnchorImported" class="stepper-content">
+        <!-- Member step -->
+        <v-stepper-content step="1">
+          <OwnerMemberStep
+            @done="ownerMemberReady"
+            :showPreviousButton="false"
+          />
+        </v-stepper-content>
+        <!-- PIN step -->
+        <v-stepper-content step="2">
+          <TokenPinStep @previous="currentStep = 1" @done="tokenPinReady" />
+        </v-stepper-content>
+      </v-stepper-items>
+
+      <v-stepper-items v-else class="stepper-content">
+        <!-- Anchor step -->
+        <v-stepper-content step="1">
+          <ConfigurationAnchorStep @done="configAnchorReady" />
+        </v-stepper-content>
+        <!-- Member step -->
+        <v-stepper-content step="2">
+          <OwnerMemberStep
+            @previous="currentStep = 1"
+            @done="ownerMemberReady"
+          />
+        </v-stepper-content>
+        <!-- PIN step -->
+        <v-stepper-content step="3">
+          <TokenPinStep
+            @previous="currentStep = 2"
+            @done="tokenPinReady"
+            :saveBusy="pinSaveBusy"
+          />
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
+
+    <!-- Confirm dialog for warnings when initializing server -->
+    <warningDialog
+      :dialog="confirmInitWarning"
+      :warnings="warningInfo"
+      localizationParent="initialConfiguration.warning"
+      @cancel="confirmInitWarning = false"
+      @accept="acceptInitWarning()"
+    ></warningDialog>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import SubViewTitle from '@/components/ui/SubViewTitle.vue';
+import TokenPinStep from './TokenPinStep.vue';
+import ConfigurationAnchorStep from './ConfigurationAnchorStep.vue';
+import WarningDialog from './WarningDialog.vue';
+import OwnerMemberStep from './OwnerMemberStep.vue';
+import { Key, Token } from '@/openapi-types';
+import { RouteName, UsageTypes } from '@/global';
+import * as api from '@/util/api';
+
+export default Vue.extend({
+  components: {
+    SubViewTitle,
+    TokenPinStep,
+    ConfigurationAnchorStep,
+    OwnerMemberStep,
+    WarningDialog,
+  },
+  props: {},
+  computed: {
+    ...mapGetters([
+      'isAnchorImported',
+      'isServerOwnerInitialized',
+      'isServerCodeInitialized',
+    ]),
+  },
+  data() {
+    return {
+      currentStep: 1,
+      pinSaveBusy: false,
+      warningInfo: [],
+      confirmInitWarning: false,
+      requestPayload: undefined as any,
+    };
+  },
+  methods: {
+    configAnchorReady(): void {
+      this.currentStep++;
+    },
+
+    ownerMemberReady(): void {
+      this.currentStep++;
+    },
```

Comment:
How about just merging these to more generic:

`
nextStep(): void {
  this.currentStep++;
}
`

Reply:
fixed